File: 3174-MinimumNumberOfChangesToMakeBinaryStringBeautiful/3174-MinimumNumberOfChangesToMakeBinaryStringBeautiful.java
// Last updated: 14/07/2026, 14:43:22
class Solution {
    public int minChanges(String s) {
        int k=0;
        for(int i=0;i<s.length();i+=2){
            if(s.charAt(i)!=s.charAt(i+1)){
                k++;
            }
        }
        return k;
    }
}

File: 3174-MinimumNumberOfChangesToMakeBinaryStringBeautiful/3174-MinimumNumberOfChangesToMakeBinaryStringBeautiful.py
# Last updated: 14/07/2026, 14:43:16
class Solution(object):
    def minChanges(self, s):
        res=0
        for i in range(0,len(s)-1,2):
            if(s[i]!=s[i+1]):
                res=res+1
        return res